wiki/en/Software-Manual.md

```diff
@@ -84,7 +84,7 @@ The Connection Setup window shows a list of available servers together with the
 
 You can filter the list by server name or location. To list only occupied servers, enter a "#" character.
 
-If you know the IP address or URL of a private server, you can connect to it using the Server Name/Address
+If you know the IP address or URL of unregistered server, you can connect to it using the Server Name/Address
```

Comment:
> If you know the IP address or URL

![image](https://user-images.githubusercontent.com/1549463/148686162-d75c5efd-44d4-481a-91e2-4a6e2733d5e2.png)

It's _not_ a URL -- it doesn't have the right format.  _All_ URLs have a protocol prefix.  We _do not_ support URLs.  Do not say that we do in the documentation...

Also the label is just "Server Address" (currently - to change to just "Address" at some point, as this screen is for connecting to a server).